Services/Simulation/JavascriptInterpreter.cs

```diff
@@ -46,25 +50,30 @@ public class JavascriptInterpreter : IJavascriptInterpreter
         /// <summary>
         /// Load a JS file and execute the main() function, passing in
         /// context information and the output from the previous execution.
-        /// Returns a map of values.
+        /// Modifies the internal device state with the latest values.
```

Comment:
I don't think it always modifies internal device state... it can though.

Reply:
I believe the call to `this.UpdateState(output);` changes this behavior. The return values is now void.